tests/tests.py

```diff
@@ -1,5 +1,6 @@
 # -*- coding: utf-8 -*-
 import pytest
+from cornice import Service
```

Comment:
I'm not opposed to testing cornice, I would maybe move it to separate file and make it optional to test, only if cornice package is in the env.